### PR TITLE
Implement design refinements to "Staff" page #6674

### PR DESCRIPTION
**Pull request by sergiogcx**
Closes https://github.com/cityofaustin/atd-data-tech/issues/6674

** UI Change only, see live demo:
https://deploy-preview-380--atd-moped-main.netlify.app/moped/staff

Changes to the staff list:
- [x] Remove pagination (show all records)
- [x] Remove "Date Added" and "Cognito user ID"*
- [x] Increase "Workgroup" column width so that it doesn't wrap at full/desktop width.
- [x] Add column "Role" (viewer, editor, or admin)
- [x] Default sort alphabetical by first name

This is the way it looks like now:

screenshot

### CODE REVIEW

File: moped-editor/src/views/staff/StaffListViewQueryConf.js [hunk 1]
```diff
@@ -114,22 +111,20 @@ export const StaffListViewQueryConf = {
...
+      label: "Role",
+      filter: value => (value ?? "N/A").replaceAll("moped-", ""),
+      width: "10%",
       type: "String",
     },
   },
-  order_by: {},
+  order_by: {
+    first_name: "asc",
+  },
```
Comment (sergiogcx) [inline moped-editor/src/views/staff/StaffListViewQueryConf.js:125]: Default sort is by first name ascending

File: moped-editor/src/views/staff/StaffListViewQueryConf.js [hunk 2]
```diff
@@ -114,22 +111,20 @@ export const StaffListViewQueryConf = {
...
-      label: "Cognito user ID",
-      search: {
-        label: "Search by Cognito User ID",
-        operator: "_eq",
-        quoted: true,
-        envelope: "{VALUE}",
-      },
+      label: "Role",
+      filter: value => (value ?? "N/A").replaceAll("moped-", ""),
+      width: "10%",
```
Comment (sergiogcx) [inline moped-editor/src/views/staff/StaffListViewQueryConf.js:119]: Shows role, removes the "moped-*" string

File: moped-editor/src/components/GridTable/GridTable.js [hunk 3]
```diff
@@ -542,12 +547,14 @@ const GridTable = ({ title, query }) => {
...
-                  setPagination={setPagination}
-                />
+                {query.config.showPagination && (
+                  <GridTablePagination
+                    query={query}
+                    data={data}
+                    pagination={pagination}
+                    setPagination={setPagination}
+                  />
+                )}
```
Comment (sergiogcx) [inline moped-editor/src/components/GridTable/GridTable.js:557]: The showPagination setting was provided but not implemented, so I just added a few extra lines of code to take care of that. Whenever showPagination is false, it disables pagination entirely and shows everything in a single page.

**Review by chiaberry — APPROVED** [review]

**Review by amenity — CHANGES_REQUESTED** [review]
@sergiogcx - thanks, this looks super. 

I think a few tweaks to column width would make the best use of space:

- First name - 15%
- Last name - 15%
- Email - 20%
- Title - 20%
- Workgroup - 20%
- Role - 10%

**Review by mateoclarke — APPROVED** [review]
🚢

**Comment by sergiogcx** [comment]
@amenity I've updated the column widths, lmk if there are any more refinements you would like to include.

**Review by amenity — APPROVED** [review]
LSTM! ✨

**Pull request merged**